src/timesigglyph.ts

```diff
@@ -69,8 +72,11 @@ export class TimeSignatureGlyph extends Glyph {
 
       this.botGlyphs.push(botGlyph);
       botWidth += defined(botGlyph.getMetrics().width);
+      height = Math.max(height, botGlyph.getMetrics().height);
     }
 
+    this.lineShift = height > 20 ? 1 : 0;
```

Comment:
Can you explain this one line for me? (And maybe add it as a comment.)

Why do we need lineShift, and why are we hard coding 20 for the height threshold, and why is the lineShift changed to 1? :-)

Reply:
The smufl spec "Digits for time signatures should be scaled such that each digit is two staff spaces tall, i.e. 0.5 em, and vertically centered on the baseline." In Vexflow a staff space is 10. 
The conditions above means: If the height of the digits is more than two staff spaces (20), shift to the next line in order to center the digits on lines 1 and 5 rather than 2 and 4.

Gonville, Leland and Bravura provide heights between 16 and 19. Petaluma more than 30.

Reply:
Comment added